airflow/gcp/operators/gcs.py

```diff
@@ -625,3 +625,36 @@ def execute(self, context: Dict):
                 object_name=self.destination_object,
                 filename=destination_file.name
             )
+
+
+class GCSDeleteBucketOperator(BaseOperator):
+    """
+    Deletes bucket from a Google Cloud Storage, either
+
+    :param bucket_name: The GCS bucket to delete from
+    :type bucket_name: str
+    """
+
+    template_fields = ('bucket_name', )
+
+    @apply_defaults
+    def __init__(self,
+                 bucket_name: str,
+                 gcp_conn_id: str = 'google_cloud_default',
+                 google_cloud_storage_conn_id: Optional[str] = None,
+                 *args, **kwargs) -> None:
+
+        if google_cloud_storage_conn_id:
+            warnings.warn(
+                "The google_cloud_storage_conn_id parameter has been deprecated. You should pass "
+                "the gcp_conn_id parameter.", DeprecationWarning, stacklevel=3)
```

Comment:
It's a new operator so no need for this deprecation, just remove `google_cloud_storage_conn_id` :)

Reply:
Aw, ok. Now I know :) Removed.